module/modmanager/manager.go

```diff
@@ -343,6 +343,11 @@ func (mgr *Manager) add(ctx context.Context, conf config.Module, moduleLogger lo
 		return nil
 	}
 
+	exists, existingName := mgr.execPathAlreadyExists(&conf)
+	if exists {
+		return errors.Errorf("An existing module %s already exists with the same executable path as module %s", existingName, conf.Name)
```

Comment:
does this break the entire reconfigure? or does the reconfigure loop continue for other modules

Reply:
It will only stop reconfigure for this module. The reconfigure loop is higher up in the stack, this method is called for each module in the case where there are multiple modules being reconfigured.

So this one module would error, and the existing module that is already configured with the same exepath will keep running.